src/util.js

```diff
@@ -14,12 +14,17 @@ const focusableElementsList = [
     '*[contenteditable]'
 ];
 
-function createTrapBoundary() {
-    const div = document.createElement('div');
-    div.setAttribute('tabindex', '0');
-    div.className = 'keyboard-trap-boundary';
+let trapBoundary;
+if (typeof document !== "undefined") {
+    trapBoundary = document.createElement('div');
+    trapBoundary.setAttribute('tabindex', '0');
+    trapBoundary.className = 'keyboard-trap-boundary';
+}
 
-    return div;
+const noop = { addEventListener: () => {} };
```

Comment:
The addEventListener seems a little cryptic to me.

Reply:
it's because `addEventListener` gets called here: https://github.com/makeup-js/makeup-keyboard-trap/pull/2/files#diff-1fdf421c05c1140f6d71444ea2b27638R28 Since it is not in a function, all of that code gets called when the module is loaded on the server. The other option is to wrap all those other `addEventListener` calls in a document "check" conditional, but it seemed maybe better to control it from the source.

Reply:
Thanks! Make sense now. Still getting my head around the server-side aspects.